src/client/snippets/paportal-liquid.code-snippets

```diff
@@ -87,5 +87,75 @@
 			"{% webform name:\"$1\" %}"
 		],
 		"description": "Fully renders a web form by name"
-	}
+	},
+    "1 Column Section": {
```

Comment:
Please rename these: Section-1-Column
Same for the prefix as well.

Reply:
Done.